src/ServiceControl/Infrastructure/Settings/Settings.cs

```diff
@@ -526,6 +526,14 @@ int GetDataSpaceRemainingThreshold()
             return threshold;
         }
 
+
+        DataStoreType GetDataStoreType()
+        {
+            var value = SettingsReader<string>.Read("DataStoreType", "DataStoreType.RavenDb");
```

Comment:
Also, the other "ConnectionString" property, uses the ConnectionStrings section of the config file as a fallback. Maybe we should be doing the same? See `Settings.GetConnectionString()` method.